tests/test_effectiveness/run_tests.py

```diff
@@ -98,30 +98,25 @@ def runbatch():
                                   mutation_rate, crossover_operator, initial_state, termination_condition,
                                   population_merge_function, iterations, population_size, number_of_children)
 
+    save_raport_path = r"C:\Users\user\PycharmProjects\CloverSolver\tests\test_effectiveness"
+    raport_name = "raport_"
+    raport_batch_max_size = 500
+    raport_iter = 0
+    raport_batch = []
     for test in tqdm(list(test_list)):
         global raport_iter
         global raport_batch
 
         raport = runsingle(*test)
         raport_batch.append(raport)
         if len(raport_batch) >= raport_batch_max_size:
-            save_raport(save_path=save_raport_path, raport=raport_batch, name=raport_name + str(raport_iter))
+            save_raport(save_path=save_raport_path, raport=raport_batch, name=raport_name + str(raport_iter) + ".pickle")
             print("Saving batch ", raport_iter)
             raport_iter += 1
             raport_batch = []
 
     save_raport(save_path=save_raport_path, raport=raport_batch, name=raport_name + str(raport_iter) + ".pickle")
 
 
-def keyboardInterruptHandler(signal, frame):
-    print("Keyboard interrupt save")
-    save_raport(save_path=save_raport_path, raport=raport_batch,
-                name=raport_name + str(raport_iter) + "_interrupted" + ".pickle")
-    exit(-1)
-
-
 if __name__ == '__main__':
-    signal.signal(signal.SIGINT,
-                  keyboardInterruptHandler)  # To make it work in PyCharm select 'Emulate terminal in output console'.
-                                            # You will find this option in settings menu in 'Run' section (next to 'Git' section)
     runbatch()
```

Comment:
Please update your develop branch and pull it into your issue branch. These changes are result of running old version